packages/sdk/src/interaction/interaction.js

```diff
@@ -149,9 +159,16 @@ export const prepAccount = (acct, opts = {}) => ix => {
   const role = opts.role
   const tempId = uuid()
 
-  if (acct.authorization && isFn(acct.authorization)) acct = {resolve: acct.authorization}
+  if (acct.authorization && isFn(acct.authorization))
+    acct = {resolve: acct.authorization}
   if (!acct.authorization && isFn(acct)) acct = {resolve: acct}
 
+  const resolve = acct.resolve
+  if (resolve)
+    acct.resolve = acct =>
```

Comment:
Multiple acct.resolve assignment

Reply:
`const resolve = acct.resolve` exists because if we addressed the function via 
https://github.com/onflow/fcl-js/blob/cd2141b78b2a6bb34a14896e4298114e369780aa/packages/sdk/src/interaction/interaction.js#L168-L172
It would become an infinite loop as it recurses itself.  Thus we need a reference to the original function instead of using `acct.resolve`